pluginconfig/pluginconfig-311.yaml

```diff
@@ -32,37 +32,37 @@ imageVersion: 311.82.20190222
 sshSourceAddressPrefixes: ['0.0.0.0/0']
 ## Container images used for specific OSA Cluster version
 images:
-  alertManagerBase: registry.access.redhat.com/openshift3/prometheus-alertmanager
-  ansibleServiceBroker: registry.access.redhat.com/openshift3/ose-ansible-service-broker:v3.11.82
+  alertManagerBase: registry.access.redhat.com/openshift3/prometheus-alertmanager:v3.11
```

Comment:
I don't understand how it can be right to specify :v3.11 anywhere here?  If we do that, then clusters can update piecemeal when the v3.11 tag is changed upstream.  Surely we need to pin it precisely as we are currently doing?  Can't everything here now read v3.11.82 ?

Reply:
This is for update testings. This way we will trigger image update.
Releases should point to particular releases, like 3.11.82, but master to "head"

Reply:
I don't think we can do this.  If we do this, we won't know for sure in dev whether our container images correspond with our VM image, or what version we're actually testing at.
We could have update/rotation testing which tries to update from v3.11.82 to v3.11 - either this is effectively a no-op but which should force a rotation, or it would test update to a newer version if it happens to exist (also with a rotation), but I don't think we can check v3.11 references into master.
Is that workable?

Reply:
Currently, all our upgrade test flow is based on tags. 
This would mean we need to change the behaviour and potentially add one more `pluginConfig-3.11.yaml.TEST` file for these values. 
So now our tests should be aware of tag (which already contains required pluginconfig) OR/AND pluginConfig file to be used. 

It is workable, but it will bring even more complexity to our testing. Its just moving the complexity from one place to another place :/

Reply:
@jim-minter rollbacked version changes, and will go for little bit different approach in separate PR